src/components/Table/TableHead/TableHead.jsx

```diff
@@ -169,24 +169,12 @@ const TableHead = ({
     });
   };
 
-  const getRenderedWidths = () => {
```

Comment:
Were you able to detect a performance improvement with the introduction of the `useCallback`? I'm not seeing where it has any effect in this scenario. This function is only called once on init and once per resize drop even without the `useCallback`. And I guess that when the table data changes, the `useLayoutEffect` won't be called unless any of the now explicit dependencies changes, meaning that `getRenderedWidths` won't be called unless needed. But perhaps I'm missing something?

Reply:
if we don't use useCallback on getRenderedWidth, the useLayoutEffect will trigger every render since the function pointer will change for getRenderedWidth

Reply:
@scottdickerson, but in practice if I count the calls to `useLayoutEffect` (using console.log) I see no difference with or without `useCallback` here. I tested a range of things like loading the component, resizing, sorting, row selection, loading more data etc. 

Sorry, just trying to understand. Please bear with me :-)